utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/wifi.lua

```diff
@@ -2,28 +2,46 @@ local ubus = require "ubus"
 local iwinfo = require "iwinfo"
 
 local function scrape()
+  local metric_wifi_radio_config = metric("wifi_radio_config","gauge")
```

Comment:
The best pratice is the `_info` suffix for this though, not `_config`. Like its wired counterpart:
```
# HELP node_network_info Non-numeric data from /sys/class/net/<iface>, value is always 1.
# TYPE node_network_info gauge
node_network_info{address="00:00:00:00:00:00",broadcast="00:00:00:00:00:00",device="lo",duplex="",ifalias="",operstate="unknown"} 1
```